Build process failing because of missing file

### Expected behavior
Build should be successful.
### Actual behavior
It's failing with message:
```
export PATH="$LISK_PATH/bin:$LISK_PATH/pgsql/bin:$PATH"
export PM2_HOME=$LISK_PATH/.pm2
# LISK_NETWORK is set at build time
export LISK_NETWORK=testnet
install --mode=0755 target/lisk.sh src/lisk-2.0.0-alpha.3-Linux-x86_64_image/lisk.sh
install --mode=0755 target/lisk_snapshot.sh src/lisk-2.0.0-alpha.3-Linux-x86_64_image/lisk_snapshot.sh
install --mode=0644 target/shared.sh src/lisk-2.0.0-alpha.3-Linux-x86_64_image/shared.sh
install: cannot stat 'target/shared.sh': No such file or directory
Makefile:112: recipe for target 'src/lisk-2.0.0-alpha.3-Linux-x86_64_image/shared.sh' failed
make: *** [src/lisk-2.0.0-alpha.3-Linux-x86_64_image/shared.sh] Error 1
```
### Steps to reproduce
Run jenkins job: `lisk-core-build-tag-private`
### Which version(s) does this affect? (Environment, OS, etc...)
`2.0.0-alpha.3`

--- a/build/Makefile
+++ b/build/Makefile
@@ -56,7 +56,6 @@ src/lisk-$(LISK_VERSION)-$(UNAME_S)-x86_64_image/finished: \
 		src/lisk-$(LISK_VERSION)-$(UNAME_S)-x86_64_image/env.sh \
 		src/lisk-$(LISK_VERSION)-$(UNAME_S)-x86_64_image/lisk.sh \
 		src/lisk-$(LISK_VERSION)-$(UNAME_S)-x86_64_image/lisk_snapshot.sh \
-		src/lisk-$(LISK_VERSION)-$(UNAME_S)-x86_64_image/shared.sh \
 		src/lisk-$(LISK_VERSION)-$(UNAME_S)-x86_64_image/tune.sh \
 		src/lisk-$(LISK_VERSION)-$(UNAME_S)-x86_64_image/bin/jq \
 		src/lisk-$(LISK_VERSION)-$(UNAME_S)-x86_64_image/bin/lisk \
@@ -108,8 +107,6 @@ src/lisk-$(LISK_VERSION)-$(UNAME_S)-x86_64_image/config.json: | src/lisk-$(LISK_
 
 src/lisk-$(LISK_VERSION)-$(UNAME_S)-x86_64_image/env.sh: | src/lisk-$(LISK_VERSION)-$(UNAME_S)-x86_64_image
 	( cat target/env.sh; echo "export LISK_NETWORK=$(LISK_NETWORK)" ) |tee $@
-src/lisk-$(LISK_VERSION)-$(UNAME_S)-x86_64_image/shared.sh: | src/lisk-$(LISK_VERSION)-$(UNAME_S)-x86_64_image
-	$(INSTALL) --mode=0644 target/$(@F) $@
 src/lisk-$(LISK_VERSION)-$(UNAME_S)-x86_64_image/%.sh: | src/lisk-$(LISK_VERSION)-$(UNAME_S)-x86_64_image
 	$(INSTALL) --mode=0755 target/$(@F) $@
 src/lisk-$(LISK_VERSION)-$(UNAME_S)-x86_64_image/etc/%: | src/lisk-$(LISK_VERSION)-$(UNAME_S)-x86_64_image/etc

